havenapi/actions/keycloak.go

```diff
@@ -0,0 +1,90 @@
+package actions
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"os"
+)
+
+type token struct {
+	Token string `json:"access_token"`
```

Comment:
Do you have a sample of the other fields that are included in the returned JSON? I wonder if we should include the others in the struct as well.

Reply:
Here ya go!
```
{
"access_token":"snip",
"expires_in":60,
"refresh_expires_in":1800,
"refresh_token":"snip",
"token_type":"bearer",
"not-before-policy":0,
"session_state":"snip",
"scope":"email profile"
}
```